src/style/common.ts

```diff
@@ -2,11 +2,34 @@
 
 import { Icon, Style } from 'ol/style'
 import continuousIcon from '../assets/icons/continuous.png'
+import spotIcon from '../assets/icons/spot.png'
+import { vcsLayerName } from '@vcmap/core'
+import { RENNES_LAYER } from '@/stores/layers'
 
-export const ContinuousMeasurementStyle: Style = new Style({
-  image: new Icon({
-    opacity: 1,
-    src: continuousIcon,
-    scale: 1,
-  }),
-})
+function getIconFromLayerName(layerName) {
+  if (layerName === RENNES_LAYER.customLayerContinuousMeasurement) {
```

Comment:
It's also good to put them in a dictionary `layerName` as the key, and the icon as the value. But this is also fine. Perhaps after having more than 2, we can switch to a dictionary for easier reading